Source/ProgressionSystemRuntime/Private/Components/PSHUDComponent.cpp

```diff
@@ -109,19 +109,11 @@ void UPSHUDComponent::SavePoints(EEndGameState EndGameState)
 void UPSHUDComponent::OnGameStateChanged_Implementation(ECurrentGameState CurrentGameState)
 {
 	CurrentGameStateInternal = CurrentGameState;
-	if (!ensureMsgf(ProgressionMenuWidgetInternal, TEXT("ASSERT: [%i] %hs:\n'ProgressionMenuWidgetInternal' is null!"), __LINE__, __FUNCTION__))
-	{
-		return;
-	}
 
 	switch (CurrentGameState)
 	{
-	case ECurrentGameState::GameStarting:
-		ProgressionMenuWidgetInternal->SetVisibility(ESlateVisibility::Collapsed);
-
 	case ECurrentGameState::Menu:
 		UpdateProgressionWidgetForPlayer();
```

Comment:
There are still plenty of places like this, where
- `break` is missing. 
- `default` does `return' instead of `break`

Please go through all `switch` in your project and fix it

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/41/commits/d933df7c6f0ab5d400df5ccd07a893521bc632fc